### PR TITLE
Add limit switches, encoder, arm commands

### PR DESCRIPTION
**Pull request by loafdog**
Add support for limit switches and encoder on arm.

Start adding commands to move arm to set positions for scoring game pieces.

This is a work in progress.  The physical arm is not complete yet.  Need to update this code once arm is fixed and lower limit switch can be used.

### CODE REVIEW

File: src/main/java/bhs/devilbotz/subsystems/Arm.java [hunk 1]
```diff
@@ -17,12 +20,25 @@
...
 
   /** The constructor for the arm subsystem. */
   public Arm() {
     armMotor =
         new CANSparkMax(
             Constants.ArmConstants.ARM_MOTOR_CAN_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
+
+    topLimitSwitch = new DigitalInput(1);
+    bottomLimitSwitch = new DigitalInput(0);
+    encoder = new Encoder(2, 3);
```
Comment (BrownGenius) [inline src/main/java/bhs/devilbotz/subsystems/Arm.java:35]: We should probably move these to constants.

**Review by BrownGenius — APPROVED** [review]
Other than my one note, looks good to me.

**Review by BrownGenius — DISMISSED** [review]
Looks good

**Review by BrownGenius — DISMISSED** [review]

**Comment by BrownGenius** [comment]
@loafdog, did you want to merge this into main?

**Comment by loafdog** [comment]
Yeah, I would like to merge this. I was hoping to test on robot today but did not happen b/c of issue with arm motor wiring.   I was tinkering with better shuffleboard setup today but haven't gotten anything great working.  Was going to poke at it tonight some more.

**Pull request merged**